mem.go

```diff
@@ -23,22 +24,53 @@ func (ca *Cagent) MemResults() (MeasurementsMap, error) {
 
 	memStat, err := mem.VirtualMemoryWithContext(ctx)
 
+	results = map[string]interface{}{
```

Comment:
For consistency I would stick with either capital or small letters in the fields. To avoid any maybe case-sensitive problems I would stick with small letters.
Also I would write out both or abbreviate both of the words. Not mix it.
If I assume correctly `B` is for bytes my preference would be `total_bytes` etc.

Reply:
You put the declaration of `results` between `memStat, err := mem.VirtualMemoryWithContext(ctx)` and the corresponding error handling. Please fix the order

Reply:
So the backend expects the field be named `metric_B`? Ok, still don't like it but if it's requirement fine.

> The reason probably that B stands for bytes, not bits
I think you rarely report bits in these cases.

Reply:
> You put the declaration of `results` between `memStat, err := mem.VirtualMemoryWithContext(ctx)` and the corresponding error handling. Please fix the order

fixed